yt/fields/field_info_container.py

```diff
@@ -421,7 +422,13 @@ def add_output_field(self, name, sampling_type, **kwargs):
         kwargs.setdefault("ds", self.ds)
         self[name] = DerivedField(name, sampling_type, NullFunc, **kwargs)
 
-    def alias(self, alias_name, original_name, units=None):
+    def alias(
+        self,
+        alias_name,
+        original_name,
+        units=None,
+        deprecate: Optional[Tuple[str]] = None,
```

Comment:
I think we've put type annotations on hold for now.

Reply:
That's not what I understood from the discussion in YTEP-0038 :/

Reply:
It is 0038 still hasn't been resolved or merged and a YTEP shouldn't be a blocker for this PR.

Reply:
I'm not saying it's a blocker, I'm specifically referring to this comment https://github.com/yt-project/ytep/pull/17#issuecomment-788188675 from @matthewturk (in response to my own)

in short: refraining ourselves from adding type annotations now in parts of the code we're already spending energy figuring out is a waste of time.

Reply:
There is a much better way to phrase this than saying "waste of time"

Reply:
I'm sorry about that, I was in a rush, which is no excuse for being rude.